app/policies/pregnancy_policy.rb

```diff
@@ -1,5 +1,5 @@
-class CasePolicy < ApplicationPolicy
+class PregnancyPolicy < ApplicationPolicy
```

Comment:
Oh definitely agree on adding it back when we need to set permissioning! I think it'll be a smidge easier to just start fresh rather than figure out what's already there.